sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -11,15 +11,10 @@
 import collections
 from typing import Any, Dict, Tuple, List, Optional, TYPE_CHECKING, cast, Union
 from datetime import timedelta
-
-try:
-    from urlparse import urlparse
-    from urllib import quote_plus  # type: ignore
-except ImportError:
-    from urllib.parse import urlparse, quote_plus
-
-from uamqp import AMQPClient, Message, authentication, constants, errors, compat, utils
+from urllib.parse import urlparse, quote_plus
 import six
+from uamqp import utils
```

Comment:
Ah, I was importing the create sas token method from uamqp.utils here since we're not adding the pyamqp library yet. Once pyamqp is merged, the plan was to use the pyamqp.utils.create_sas_token method. However, I can just add the pyamqp generate sas token method to _utils.py file here for now.